src/nidigital/metadata/enums.py

```diff
@@ -269,6 +269,7 @@
                 },
                 'name': 'NIDIGITAL_VAL_0',
                 'python_name': 'ZERO',
+                'pretty_name': '0',
```

Comment:
Please make sure the metadata changes are reflected in the internal metadata component.

Reply:
https://ni.visualstudio.com/DefaultCollection/DevCentral/_git/shelves/pullrequest/124134